pytd/writer.py

```diff
@@ -542,7 +591,9 @@ def _write_msgpack_stream(self, items, stream):
                     mp = packer.pack(normalized_msgpack(item))
                 gz.write(mp)
 
-        stream.seek(0)
```

Comment:
did this `#seek` become unnecessary?

Reply:
Yes, it's moved to here https://github.com/treasure-data/pytd/pull/134/files#diff-7e3490636bfbd0197f47ef9694662f2621beb294fb178b5185c8f257082c53e7R533
This is because to get file size by`stream.tell()` properly, `#seek` should happen later.